test/rules/completed-docs/defaults/tsconfig.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
an empty file looks pretty weird; can we at least make it extend some common tsconfig file? perhaps located at `test/rules/test-tsconfig.json`?